birdhouse/FreeCad.py

```diff
@@ -0,0 +1,140 @@
+width = 120
+extendedWidth = width*1.1
+height = 85
+thickness = 2.0
+
+def makeSketch(sketchName, plane = 'YZ_Plane', points=[], constraints=[]):
```

Comment:
This function was my main attempt at abstracting nuts and bolt code to instead taking descriptive constraints in the form of points/vertices and a list of constraints.